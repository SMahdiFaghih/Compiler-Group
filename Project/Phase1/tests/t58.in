a___________________________________________________________________________2 - 16b