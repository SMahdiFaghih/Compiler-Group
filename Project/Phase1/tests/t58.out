T_ID a___________________________________________________________________________2
-
T_INTLITERAL 16
T_ID b